TFT/GlobalSettings.txt

```diff
@@ -27,8 +27,6 @@ teleport_radius	800	units will stay within this radius from a hero that is using
 gold_exchange_level	650	computers give extra gold above this value to other computers on the same team in need of gold
 wood_exchange_level	350	computers give extra lumber above this value to other computers on the same team in need of lumber
 max_towns	5	how many towns is a computer allowed to have?
-add_tag	true	Add the [AMAI] tag in front of name

```

Comment:
I'm not going to remove these as then these settings are no longer available to change in the developer UI tool for those that use it.